telemetry/definitions/commonDefinitions.json

```diff
@@ -1168,6 +1173,28 @@
             ],
             "passive": true
         },
+        {
+            "name": "toolkit_showNotification",
+            "description": "The toolkit tried to retrieve and show notification message",
+            "metadata": [
+                {
+                    "type": "result",
+                    "required": true
+                },
+                {
+                    "type": "notificationId",
+                    "required": true
+                },
+                {
+                    "type": "reason",
+                    "required": false
+                },
+                {
+                    "type": "url",
```

Comment:
Is a URL necessary if notification Id represents the message?

Reply:
yeah i suppose its unnecessary - will remove it